docs/BasicCommunication/OnSystemCapabilityUpdated/index.md

```diff
@@ -18,10 +18,11 @@ Purpose
     * `menuName` and `templateTitle` should be included, if they are visible on HMI.
     * `mainField1-4`, `statusBar` and `mediaTrack` should not be included if OEM implemented the `WEB_VIEW` template without these text fields.
     * `availabletemplates`, `buttonCapabilities` and `imageTypeSupported` should reflect the general capabilities of the window/system.
+5. HMI may send `KeyboardCapabilities` within OnSystemCapabilityUpdated notification with the supported keyboards in `supportedKeyboards`.
```

Comment:
```suggestion
```
I don't think this note is needed, as this is just part of the existing displayCapabilities, and there is no way for the keyboard capabilities to change during runtime as-is.